app/controller.py

```diff
@@ -67,6 +67,12 @@ def get_all_items():
     itemList = items.find({'type': 'story'}, sort=[('id', pymongo.ASCENDING)])
     return itemList
 
+def get_all_items_limited(row_from,row_to):
+    items = db_con.items
+    print('Trying getting limited items')
+    itemList = items.find({'type': 'story'}, sort=[('id', pymongo.ASCENDING)]).skip(int(row_from)).limit(int(row_to))
```

Comment:
@Ansty93 I think it would make more sense to sort by the added date, as some of the IDs are randomly generated

Reply:
U are making a point, i thought it was already sorting by adding date. I'll refactor

Reply:
Done